components/ui/AppLink.vue

```diff
@@ -19,6 +23,8 @@ import MenuLinkMixin from '~/mixins/menuLink'
 export default class extends Mixins(MenuLinkMixin) {
```

Comment:
Thanks to the unification, `MenuLinkMixin` is no longer needed, all the API regarding links can live under this component. Would you mind to move its methods here and remove it, please?